Update how-to-test-attack-path-and-security-explorer-with-vulnerable-… (#116911)
Azure/azure-content: articles/defender-for-cloud/how-to-test-attack-path-and-security-explorer-with-vulnerable-container-image.md

        az acr import --name $MYACR --source DCSPMtesting.azurecr.io/mdc-mock-0001 --image mdc-mock-0001
        ```

>     1.  Create a new AKS cluster and integrate with an existing ACR:

Reviewer: If
```suggestion
    1.  If you don't have an AKS cluster, you can use the below command create a new AKS cluster:
```

>     1.  If you don't have an AKS cluster, you can use the below command create a new AKS cluster:

Reviewer 2: ```suggestion
    1.  If you don't have an AKS cluster, use the following command to create a new AKS cluster:
```
